src/components/rangeslider/attributes.js

```diff
@@ -0,0 +1,48 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var colorAttributes = require('../color/attributes');
+
+module.exports = {
+    visible: {
+        valType: 'boolean',
+        dflt: false,
```

Comment:
A blank range slider ( `layout.xaxis.rangeslider = {}` ) should be considered *visible*.